neovim/api/nvim.py

```diff
@@ -115,6 +139,12 @@ def _to_nvim(self, obj):
             return ExtType(*obj.code_data)
         return obj
 
+    def _get_lua_helpers(self):
```

Comment:
How about `_get_private_lua` or just `_get_lua`. And doc like this:

```
"""
Returns a wrapper to a private Lua module associated with this python client, providing these functions:
...
"""
```

The word "helpers" needs to go away in all of our projects, it is useless and seems to proliferate as soon as it's used once.